app/src/main/java/org/sil/gatherwords/EntryActivity.java

```diff
@@ -157,7 +157,7 @@ private void stopRecording() {
         mRecorder.stop();
         mRecorder.release();
         mRecorder = null;
-        // TODO: Store the audio file in the DB (or copy file to a permanent location and store a link).
+        // TODO: Store the audio file in the DB (or copy file to a permanent gps and store a link).
```

Comment:
If you want to use find and replace, please be careful your results are sensible.